types.cc

```diff
@@ -2360,6 +2360,14 @@ struct from_string_visitor {
     sstring_view s;
     bytes operator()(const reversed_type_impl& r) { return r.underlying_type()->from_string(s); }
     template <typename T> bytes operator()(const integer_type_impl<T>& t) { return decompose_value(parse_int(t, s)); }
+    bytes operator()(const ascii_type_impl&) {
```

Comment:
just out of curiosity: before you added this overload, ascii_type_impl was falling into string_type_impl overload, right?

Reply:
This is how I understood the code at least. `ascii_type_impl` inherits from 'string_type_impl`, so that's how I'd expect this to behave